deploy/acm-policies/50-GENERATED-rbac-permissions-operator-config-sp.Policy.yaml

```diff
@@ -211,15 +211,15 @@ spec:
                         apiVersion: rbac.authorization.k8s.io/v1
                         kind: RoleBinding
                         metadata:
-                            name: dedicated-admins-core-ns-0
+                            name: dedicated-admin-serviceaccounts-core-ns-0
```

Comment:
Please revert changes to these directories.

Reply:
in all the occurrences ?

Reply:
Yeah these changes look unrelated to your actual PR, yes?

Reply:
To give some context: the changes were done because https://ci.ext.devshift.net/job/openshift-managed-cluster-config-gh-pr-check/4440/ was lamenting that after running make there were uncommitted changes and were instructing to run make and amend the PR...

Reply:
@cblecker please advice on how to proceed. I can revert all changes but the amend to the regexp, knowing that the jenkins test will fail. Up to you.

Reply:
If you rebase this PR and run make, it should look better

Reply:
done